clang/test/CodeGen/X86/avx10_2_512ni-builtins.c

```diff
@@ -0,0 +1,24 @@
+// RUN: %clang_cc1 -flax-vector-conversions=none -ffreestanding %s -triple=x86_64-unknown-unknown -target-feature +avx10.2-512 -emit-llvm -o - | FileCheck %s
+
```

Comment:
pls add i386 test.

Reply:
is `-flax-vector-conversions=none` required?

Reply:
Added i386 test. `-flax-vector-conversions=none` is required. It is used to check no implicit vector type conversion.